### ISSUE TITLE
Suggestion: (BSL) Shader Support (Colored Planets, height appearance), Fabric 1.20.4

### ISSUE THREAD

**Issue by EfficientStinkbug** [Nettakrim/Spyglass-Astronomy#37]
Hello, 

I don't even know if it's possible or not too much effort to fix, but with BSL shaders, these things don't work:

- Colored planets/comets
- All the stars/planets are visible at night, when you're on the ground level. 

I'll just leave it here, maybe this'll be fixable in the future.
Thank you for your great mod!!

**Comment by Nettakrim** [comment]
yeah i am aware of this, i dont really have any idea how i would fix it lol

**Comment by EfficientStinkbug** [comment]
Lol, no worries. I guess this would only work if the guys from the shaders implemented this, so there's probably not much you can do from your side.

**Comment by Picklejarly** [comment]
> yeah i am aware of this, i dont really have any idea how i would fix it lol

Fairly certain it's because most shaders add their own stars and drastically change how the night sky looks. Honestly though, it still works well enough.

**Comment by setzke** [comment]
In the discord for BSL Shaders, they said the shader skybox is essentially mandatory. I don't know how the stars and planets are rendered, but would it be possible to put them IN FRONT of the shader sky? Big sad, this.
